### ISSUE TITLE
[UI Polish] Login Active State "Offline Mode" link appearance

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop-client#1245]
## Problem
When struggling with lengthy server-pings in difficult authentication situations, a user may desire abandoning ship to simply enter offline mode. However, that is not currently possible (sidenote: eventually, it would be nice to enable that—while showing the error message, still, on the login window that would then sit behind the Client window, in a separate pane).

## Solution
When in `active state`, the Login screen should show the Offline State hyperlink at `.4 alpha` of its 100% value

**Comment by Er1kkkaaa** [comment]
Hello, I am interested in starting my contributions to the project from this specific issue. I would appreciate it however if you could provide me some information about what is considered to be an active state in this case. Are we talking about the active window?

**Comment by eloquence** [comment]
I think the original issue description refers to the case where, _while you are logging in_, the login button changes to the "Signing In" state (AKA the active state) and all other form elements (including the offline mode link) are disabled, but the offline mode link is still styled as if it were clickable. Changing the alpha in this case will make it more obvious that it's not clickable.

@ninavizz Can you clarify if this is understanding is correct?

Just a note for @Er1kkkaaa that this is difficult to see outside Qubes because the "Signing In" state is very brief when testing against a local Docker environment. But you could insert sleeps in the code to see it.

**Comment by ninavizz** [comment]
@creviera @Er1kkkaaa Yup, @eloquence is totally correct. My concern, is that with my (fabulous!) wonky internet connection at home, I'm often "stuck" in the sign-in active state. As an end user, it is then tempting to just be all like "Eeh, eff it—gonna go offline..." which I have tried to do, and it cannot do. So, if in fact that is not an option (boy it would be nice if it could be!) it should more clearly be disabled.

Erik, is it on some TBD backlog dealie to be able to click on "Use Offline" as a hybrid command to both cancel an authentication, and enter the client offline?
